aqua/ipfs.aqua

```diff
@@ -28,6 +28,9 @@ service Ipfs("aqua-ipfs"):
   cat(hash: string) -> IpfsCatResult
   cat_from(hash: string, external_multiaddr: string) -> IpfsCatResult
   connect(multiaddr: string) -> IpfsResult
+  dag_get(hash: string) -> IpfsGetResult
+  dag_get_from(hash: string, external_multiaddr: string) -> IpfsGetResult
+  dag_put(file_path: string) -> IpfsPutResult
```

Comment:
is that from vault or how?

Reply:
yeah, it's the same as existent `put(file_path: string) -> IpfsPutResult`